pkg/build/strategies/sti/sti.go

```diff
@@ -306,9 +306,13 @@ func (builder *STI) PostExecute(containerID, location string) error {
 }
 
 func (builder *STI) getPreviousImage() string {
+	if !builder.incremental || !builder.config.RemovePreviousImage {
+		return ""
+	}
```

Comment:
maybe we shouldn't even be calling this if we're not doing an incremental build?  it seems weird to return "".